packages/manager/apps/billing/src/app.module.js

```diff
@@ -0,0 +1,44 @@
+import 'script-loader!moment/min/moment.min.js'; // eslint-disable-line
+import angular from 'angular';
+import uiRouter from '@uirouter/angularjs';
+import 'bootstrap';
+import 'angular-ui-bootstrap';
+import ngPaginationFront from '@ovh-ux/ng-pagination-front';
+
+import { Environment } from '@ovh-ux/manager-config';
+import ovhManagerCore from '@ovh-ux/manager-core';
+import { detach as detachPreloader } from '@ovh-ux/manager-preloader';
+import ngOvhUiRouterLineProgress from '@ovh-ux/ng-ui-router-line-progress';
+import ngUiRouterBreadcrumb from '@ovh-ux/ng-ui-router-breadcrumb';
+import ngQAllSettled from '@ovh-ux/ng-q-allsettled';
+
+import { billingManagement } from '@ovh-ux/manager-billing';
+
+Environment.setVersion(__VERSION__);
+
+const moduleName = 'BillingApp';
+
+angular
+  .module(moduleName, [
+    ovhManagerCore,
+    ngOvhUiRouterLineProgress,
+    ngUiRouterBreadcrumb,
+    uiRouter,
+    billingManagement,
+    ngQAllSettled,
+    'ui.bootstrap',
+    ngPaginationFront,
+  ])
+  .config(
+    /* @ngInject */ ($locationProvider) => $locationProvider.hashPrefix(''),
+  )
```

Comment:
Maybe we can have a redirection to the default route to avoid landing on a blank page